<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewDogWindow</class>
 <widget class="QWidget" name="ViewDogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/images/logo-icon.png</normaloff>../resources/images/logo-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F8F3E0; </string>
  </property>
  <widget class="QFrame" name="navigationFrame">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>34</y>
     <width>1158</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  #E9E4D4;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="navigationMenuLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>18</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #017F36; 
height: 120px;
font-weight:  600;
font-family: Montserrat;
font-size: 20px;
color: #fff;</string>
    </property>
    <property name="text">
     <string>Doniraj</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>341</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="homeLabelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #017F36;
font-size: 22px;
font-family: Montserrat;</string>
       </property>
       <property name="text">
        <string>Početna</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="aboutUsLabelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #017F36;
font-size: 22px;
font-family: Montserrat;</string>
       </property>
       <property name="text">
        <string>O nama</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="contactUsLabelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #017F36;
font-size: 22px;
font-family: Montserrat;</string>
       </property>
       <property name="text">
        <string>Kontakt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="pawIconOnTheEdge_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>680</y>
     <width>132</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 20%;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>412</width>
     <height>542</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #D9D9D9;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="dogsImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>452</y>
      <width>415</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(233,228,212, 0.9); border-top-left-radius: 0px !important; border-top-right-radius: 0px !important; border-bottom-left-radius: 20px !important; border-bottom-right-radius: 20px !important;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="dogsName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #017F36;
font-family: Montserrat;
font-weight: bold;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="dogsShortDescription">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #444;
font-family: Montserrat;
font-weight: 400;
font-size: 18px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>811</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E9E4D4;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="backToTheDogsListButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>17</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoDogsImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>319</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>145</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: bold;
font-size: 26px;</string>
    </property>
    <property name="text">
     <string>Datum rođenja:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>182</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: bold;
font-size: 26px;</string>
    </property>
    <property name="text">
     <string>Težina:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>212</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: bold;
font-size: 26px;</string>
    </property>
    <property name="text">
     <string>Visina:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>235</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: bold;
font-size: 26px;</string>
    </property>
    <property name="text">
     <string>Broj vakcina:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>265</y>
      <width>361</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: bold;
font-size: 26px;</string>
    </property>
    <property name="text">
     <string>Zadnja posjeta veterinaru:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deleteDogButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>190</width>
      <height>58</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="editDogButton_1">
    <property name="geometry">
     <rect>
      <x>477</x>
      <y>150</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="editDogButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>183</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="editDogButton_3">
    <property name="geometry">
     <rect>
      <x>408</x>
      <y>215</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="editDogButton_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>243</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="editDogButton_5">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>270</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="birthDateLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>145</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: 400;
font-size: 26px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:26px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26px;&quot;&gt;2017-04-25&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="weightLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>178</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: 400;
font-size: 26px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:26px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26px;&quot;&gt;30kg&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: 400;
font-size: 26px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:26px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26px;&quot;&gt;30cm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vaccineNumberLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: 400;
font-size: 26px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:26px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26px;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lastVetVisitLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>265</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-family: Montserrat;
font-weight: 400;
font-size: 26px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:26px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26px;&quot;&gt;2017-04-25&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="cancelEditButton_1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cancelEditButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>183</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cancelEditButton_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>215</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cancelEditButton_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>243</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cancelEditButton_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>23</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>birthDateLabel</zorder>
   <zorder>heightLabel</zorder>
   <zorder>weightLabel</zorder>
   <zorder>vaccineNumberLabel</zorder>
   <zorder>lastVetVisitLabel</zorder>
   <zorder>backToTheDogsListButton</zorder>
   <zorder>infoDogsImage</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>deleteDogButton</zorder>
   <zorder>editDogButton_1</zorder>
   <zorder>editDogButton_2</zorder>
   <zorder>editDogButton_3</zorder>
   <zorder>editDogButton_4</zorder>
   <zorder>editDogButton_5</zorder>
   <zorder>cancelEditButton_1</zorder>
   <zorder>cancelEditButton_2</zorder>
   <zorder>cancelEditButton_3</zorder>
   <zorder>cancelEditButton_4</zorder>
   <zorder>cancelEditButton_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
